Tools/Scripts/webkitpy/layout_tests/controllers/manager.py

```diff
@@ -377,7 +377,7 @@ def run(self, args):
         # Create the output directory if it doesn't already exist.
         self._port.host.filesystem.maybe_make_directory(self._results_directory)
 
-        needs_http = any(test.needs_http_server for tests in itervalues(tests_to_run_by_device) for test in tests)
+        needs_http = (any(test.needs_http_server for tests in itervalues(tests_to_run_by_device) for test in tests) or self._options.load_in_cross_origin_iframe)
```

Comment:
Is this part needed? I think this should already work after https://commits.webkit.org/276514@main

Reply:
Checking the code, it seems driver.is_http_test only affects the url to be used; test.is_http_test decides whether server will start